install_config/install/deploy_router.adoc

```diff
@@ -1061,18 +1061,19 @@ $ oadm policy add-scc-to-user privileged -z router
 
 Now that the router use can create privileged containers, make the
 router deployment configuration use the power so that the router can
-set the iptables rules it needs.  This patch changes the router
-deployment configuration so that the containter that is created runs
+set the iptables rules it needs. This patch changes the router
+deployment configuration so that the container that is created runs
 as root.
+
 ----
 $ oc patch dc router -p '{"spec":{"template":{"spec":{"containers":[{"name":"router","securityContext":{"privileged":true}}]}}}}'
 ----
 
 *Tell the Router to Use iptables*
 
-Set the option on the router deployment configuration.  If you used a
+Set the option on the router deployment configuration. If you used a
```

Comment:
I'd put a colon after "configuration", then the example command, and then put the 2nd sentence after the example, like a footnote.